Core/src/commonMain/kotlin/com/sdk/growthbook/network/NetworkDispatcher.kt

```diff
@@ -25,4 +25,6 @@ interface NetworkDispatcher {
         onSuccess: (String) -> Unit,
         onError: (Throwable) -> Unit
     )
+
+    fun setLoggingEnabled(enabled: Boolean)
```

Comment:
We already have compiled binaries with the implementation of NetworkDispatcher. For me using current version of OkHttp dispatcher causes runtime error `java.lang.AbstractMethodError`. I'll rewrite this code without changing NetworkDispatcher interface but from my curiosity can you advise something in this case. Seems like default implementation doesn't help some why

Reply:
@JvmDefault annotation is not available in commonMain source set